package.json

```diff
@@ -100,6 +101,7 @@
     "json-refs": "^2.1.5",
     "jwt-decode": "^2.2.0",
     "lodash": "^4.13.1",
+    "md5": "^2.2.1",
```

Comment:
This module should be remove since it has 3 additional dependencies which are downloaded.

The `md5` usage can be replaced with something like this:

```javascript
const crypto = require('crypto');

const hash = crypto.createHash('sha256').update('foo bar').digest('hex');

console.log(hash);
```

This uses `sha256` instead of `md5` since `md5` is broken and has known collisions.